README.md

```diff
@@ -13,27 +13,30 @@ Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
 
 Installing
 ----------
-Releases are made on the **stable** branch. To install the latest release:
 
-    % git clone https://github.com/STScI-JWST/jwst.git
+To install the latest stable version of the library, we recommend using [conda](https://conda.io/docs/index.html) and
+the [astroconda](https://astroconda.readthedocs.io) channel:
 
-    % cd jwst
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
+    conda install jwst
 
-    % git checkout stable
+To install the development version of the repository, we recommend using the [astroconda](https://astroconda.readthedocs.io) channel
+to install the dependencies, and then installing from the github repository:
 
-    % python setup.py install
+    conda config --add channels http://ssb.stsci.edu/astroconda-dev
```

Comment:
I recall adding dev to the channels was a no-no back when we first set all this up.  Is that still the case @jhunkeler?

Reply:
A bunch of this info is in the **Conda Dev and You** document on Confluence, but I agree it the basics of how to install and update should be here in the repository.